spec/truefi2/BorrowingMutex.spec

```diff
@@ -22,11 +22,10 @@ rule onePlusTwoEqualsThree(uint one, uint two) {
     assert three == 3, "One plus two does not equal three";
 }
 
-rule functionDoesNotBanUnlockedBorrower() {
+rule functionDoesNotBanUnlockedBorrower(method f) {
```

Comment:
This change serves two purposes:
1. This allows the use of `filtered` keyword (mentioned in #1130)
2. This divides variables nicely into two categories - `method f` works differently than `address alice` in that the former generates an extra rule for each possible method, while the latter translates into "for any address alice" in the underlying SMT solver.